inc/Addon/Varnish/Varnish.php

```diff
@@ -111,8 +111,23 @@ public function purge( $url ) {
 		);
 
 		foreach ( $varnish_ip as $ip ) {
-			$host      = ! empty( $ip ) ? $ip : str_replace( '*', '', $parse_url['host'] );
-			$purge_url = $scheme . '://' . $host . $parse_url['path'] . $regex;
+			$host           = ! empty( $ip ) ? $ip : str_replace( '*', '', $parse_url['host'] );
+			$purge_url_main = $scheme . '://' . $host . $parse_url['path'];
+
+			/**
+			 * Filters the purge url.
+			 *
+			 * @since 3.6.3
+			 *
```

Comment:
One parameter is missing for the first one containing the full URL

Reply:
Really you are awesome.
Done